src/transformers/__init__.py

```diff
@@ -1154,6 +1155,7 @@
     _import_structure["models.pix2struct"].extend(["Pix2StructImageProcessor"])
     _import_structure["models.poolformer"].extend(["PoolFormerFeatureExtractor", "PoolFormerImageProcessor"])
     _import_structure["models.pvt"].extend(["PvtImageProcessor"])
+    _import_structure["models.rt_detr"].extend(["RTDetrImageProcessor"])
```

Comment:
```suggestion
    _import_structure["models.rt_detr"].append("RTDetrImageProcessor")
```
(nit)

Reply:
@NielsRogge All the other uses extend instead of append. Do we have to use append?

Reply:
Ok, you can keep it this way